app/Http/Controllers/CSVController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\TransactionBuilder;
+use App\Transaction;
+
+class CSVController extends Controller
+{
+    public function index()
+    {
+        return view('csv.csvupload');
+    }
+
+    public function upload(Request $request)
+    {
+        $file = $request->file('inputFile');
```

Comment:
It might be advisable to confirm that the file is a csv file, you can do this via validation like so:

```php
$request->validate([
    'fileUpload' => 'required|mimes:csv',
]);
```